docs/installation-and-operation/serialization.md

```diff
@@ -4,53 +4,229 @@ redirect_from:
+table_id: null
+result_metadata: null
+database_id: Sample Database
+enable_embedding: false
+collection_id: onou5H28Wvy3kWnjxxdKQ
+query_type: native
+name: Plan counts by source
+creator_id: jeff@metabase.com
+made_public_by_id: null
+embedding_params: null
+cache_ttl: null
+dataset_query:
+  type: native
+  native:
+    query: |-
+      SELECT count(*),
+             PLAN,
+             SOURCE
+      FROM accounts
+      WHERE SOURCE IS NOT NULL
+      GROUP BY PLAN,
+               SOURCE
+    template-tags: {}
+  database: Sample Database
+parameter_mappings: []
+serdes/meta:
+- model: Card
+  id: 17p_H8e2OpHGGJVxqg4sN
+  label: plan_counts_by_source
+display: table
+entity_id: 17p_H8e2OpHGGJVxqg4sN
+collection_preview: true
+visualization_settings:
+  table.pivot_column: PLAN
+  table.cell_column: COUNT(*)
+  column_settings: null
+parameters: []
+dataset: false
+created_at: '2023-05-22T14:32:28.124325'
+public_uuid: null
+```
+
+## Before exporting or importing
+
+If your Metabase is:
+
+- currently running, and
+- you're using the default H2 database
+
+You'll need to stop Metabase before exporting or importing.
+
+Otherwise, you're good.
+
+## Exporting a Metabase
+
+To export the contents of a Metabase instance, use the following command in your terminal:
+
+```
+java -jar metabase.jar export [export_name]
+```
+
+## Export options
+
+To view a list of `export` options, use the `help` command:
+
+```
+java -jar metabase.jar help export
+```
+
+Which will print something like:
+
+```
+export path & options
+	 Serialize Metabase instance into directory at `path`.
+	 Options:
+	   -u, --user EMAIL                Include collections owned by the specified user
+	   -c, --collection ID             Export only specified ID; may occur multiple times.
+	       --collections ID_LIST       (Legacy-style) Export collections in comma-separated list of IDs, e.g. '123,456'.
+	   -C, --no-collections            Do not export any content in collections.
+	   -S, --no-settings               Do not export settings.yaml
+	   -D, --no-data-model             Do not export any data model entities; useful for subsequent exports.
+	   -f, --include-field-values      Include field values along with field metadata.
+	   -s, --include-database-secrets  Include database connection details (in plain text; use caution).
+```
+
+### `--user`
+
+The `user` flag tells Metabase to include collections owned by the specified user, identified by email. QUESTION: What about accounts created with SSO?
+
+### `--collection`
+
+The `--collection` flag (alias `-c`) lets you specify by ID one or more collections to include in the export. You can find the collection ID in the collection's URL, e.g., for a collection at: `your-metabase.com/collection/42-terraforming-progress`, the ID would be `42`.
+
+If you want to specify multiple collections, separate the IDs with commas. E.g.,
+
+```
+java -jar metabase.jar export --collection 1,3,5,9
+```
+
+### `--no-collection`
```

Comment:
This doesn't let you specify collections. It is an all or nothing command. If you add -C no collection is serialized.